packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -156,6 +158,23 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
   );
 }
 
+async function updateProfilePhoto(oAuth2Client: Auth.OAuth2Client, userId: number) {
+  try {
```

Comment:
@zomars this function is already wrapped in a try/catch block so even if it errors it'll still create the calendar credentials